{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Test defects","children":[{"name":"selenium.common.exceptions.WebDriverException: Message: Process unexpectedly closed with status 1","children":[{"name":"test_send_and_delete_message_1_with_attachment[firefox]","uid":"890e2288a84f9ad5","parentUid":"359adc3b20c3cd6e289fa31b85222924","status":"broken","time":{"start":1730129789577,"stop":1730129789577,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":["'firefox'"],"tags":["order(1)","@pytest.mark.usefixtures('setup')"]},{"name":"test_send_message_2_add_remove_reactions_and_cleanup[firefox]","uid":"e43264729f600594","parentUid":"359adc3b20c3cd6e289fa31b85222924","status":"broken","time":{"start":1730129790648,"stop":1730129790648,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":["'firefox'"],"tags":["order(2)","@pytest.mark.usefixtures('setup')"]}],"uid":"359adc3b20c3cd6e289fa31b85222924"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}